### DISCUSSION TITLE
What is the wildcard syntax in PAC?

### DISCUSSION THREAD

**Discussion by Leopold702** [shadowsocks/shadowsocks-windows#3297] [Q&A] [answered]
I want to add a rule to the PAC that will match all subdomains of a particular domain.

For example, I need a rule that will match the below domains:

- abc.domain.com
- xyz.doman.com
- 123.domain.com

I've tried several options containing wildcard character, for example:   `"||*.domain.com"` or `"||/*.domain.com/"`, but none of them worked.

I believe the wildcard items should be supported by Shadowsocks, so can someone please explain, what is the correct syntax for this?

**Comment by chenshaoju** [comment, marked as answer]
sure, ` ||example.com ` is match ` sub1.example.com `, ` ||sub.example.com` is match ` sub2.sub.example.com `.

for more info: https://adblockplus.org/en/filter-cheatsheet#blocking2
